requirements.txt

```diff
@@ -4,8 +4,7 @@ msgpack>=0.5.2
 # Currently dask and numpy==1.16.0 clash
 numpy!=1.15.0,!=1.16.0
 pandas>=0.23.0
-# pyarrow==0.14.0 breaks kartothek
-pyarrow>=0.11.1, !=0.14.0, <0.14.1 # Keep upper bound pinned until we see non-breaking releases in the future
+pyarrow
```

Comment:
The pinnings are required to be there. The packaged wheels use this information to communicate the compatibility to the users